src/test/java/org/broadinstitute/hellbender/tools/spark/validation/CompareDuplicatesSparkIntegrationTest.java

```diff
@@ -53,23 +63,15 @@ public void identicalBamTest(File firstBam, File secondBam) throws Exception {
     }
 
     @Test(dataProvider = "CompareDifferentDuplicatesProvider", expectedExceptions = UserException.class, groups = "spark")
```

Comment:
It would be nice if this threw a different Exception type than the error conditions do so that it's clear that it's throwing because it found diffs, not because the input was wrong somehow.